Stripe/STPEphemeralKey.m

```diff
@@ -73,14 +79,11 @@ - (BOOL)isEqual:(id)object {
     if (!object || ![object isKindOfClass:self.class]) {
         return NO;
     }
-    return [self isEqualToResourceKey:object];
+    return [self isEqualToEphemeralKey:object];
 }
 
-- (BOOL)isEqualToResourceKey:(STPEphemeralKey *)other {
-    return [self.stripeID isEqualToString:other.stripeID] 
-        && [self.secret isEqualToString:other.secret]
```

Comment:
why were these removed?

Reply:
@jack-stripe ?

Reply:
I renamed it to `isEqualToEphemeralKey` ("Resource Key" was an early and terrible name for ephemeral keys)

Reply:
ah sorry, I mean why was it changed so it only compares the stripeIDs?

Reply:
ping on this @jack-stripe ? (not going to block patch because id equivalence makes sense as sufficient to me but would be useful to know the reasoning

Reply:
Oh, sorry, missed your comment @csabol-stripe!
I honestly don't remember why I changed that. I don't see a world in which `stripeID` is the same but `secret` is different. I'm happy with either implementation.